### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by Galithil**
Added a lot of mails, and completed a bunch of sprint milestones

### CODE REVIEW

File: ngi_pipeline/conductor/launchers.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import importlib
 import os
+import inspect
```
Comment (mariogiov) [inline ngi_pipeline/conductor/launchers.py]: We can remove this

File: ngi_pipeline/utils/communication.py [hunk 2]
```diff
@@ -15,17 +15,21 @@ def mail(recipient, subject, text, origin="ngi_pipeline"):
...
         recipient="ngi_pipeline_operators@scilifelab.se",
-        subject="sample_analysis intervention needed", origin="ngi_pipeline", exception_text=None):
-
-    text="""
-    Project {} / Sample {} / Workflow {}
-
-    This sample analysis has encountered an error.
-    Not launching any new analysis.
+        subject="analysis intervention needed", origin="ngi_pipeline", info_text=None):
+    tb_info=traceback.extract_stack(limit=2)[-2]
```
Comment (mariogiov) [inline ngi_pipeline/utils/communication.py]: http://lmgtfy.com/?q=tb+info

**Comment by guillermo-carrasco** [comment]
just make sure that all the mails you guys send are relevant, I can tell you by experience that if they're not, they will end up behind some filter :wink:

**Comment by mariogiov** [comment]
Man, thinking about frequency and relevance of emails is above my pay grade. I suppose I could add a randomizer to the smtp headers to make the mails very difficult to filter, though.

**Pull request merged**